conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -396,6 +398,7 @@ func (algodImp *algodImporter) Close() error {
 	if algodImp.cancel != nil {
 		algodImp.cancel()
 	}
+	algodImp.logger.Infof("importer algod.Close() at round %d", algodImp.round)
```

Comment:
I don't think we need to save the round in importer. we can print the round  in the exporter.

Reply:
I wanted to introduce the round purely for logging purposes  -to know from algod importer's perspective- which round it was on. But this could be overkill. I see several approaches:
* keep as is
* revert and get rid of the round
* rename `algodImp.round` &rarr; `algodImpl.loggingRound` to clarify that it's not actually useful for anything
I don't have a strong opinion here.